insights/specs/default.py

```diff
@@ -391,6 +391,7 @@ def semid(broker):
     messages = simple_file("/var/log/messages")
     metadata_json = simple_file("metadata.json", context=ClusterArchiveContext, kind=RawFileProvider)
     mlx4_port = simple_command("/usr/bin/find /sys/bus/pci/devices/*/mlx4_port[0-9] -print -exec cat {} \;")
+    modinfo_i40e = simple_command("/sbin/modinfo i40e")
```

Comment:
You missed the `simple_file` in `insights_archive.py`

Reply:
Done